python/ray/cross_language.py

```diff
@@ -55,6 +56,7 @@ def get_function_descriptor_for_actor_method(
                                   f"not support language {language}")
 
 
+@PublicAPI
```

Comment:
How stable are these cross language APIs? Do we really want to guarantee that these won't change?

Reply:
Can change

Reply:
Doesn't that mean they should be `@PublicAPI(stability="beta")`?

Just to clarify, `stability` here refers to the likelihood of the API changing and not to the robustness of the implementation, right?